packages/combobox/stories/combobox.stories.ts

```diff
@@ -15,57 +15,61 @@ import { html, TemplateResult } from '@spectrum-web-components/base';
 import { ComboboxOption } from '..';
 import '@spectrum-web-components/combobox/sp-combobox.js';
 import '@spectrum-web-components/combobox/sp-combobox-item.js';
+import '@spectrum-web-components/field-label/sp-field-label.js';
+import '@spectrum-web-components/help-text/sp-help-text.js';
 import '@spectrum-web-components/tooltip/sp-tooltip.js';
 
 export default {
     title: 'Combobox',
     component: 'sp-combobox',
 };
 
+const optionsA: ComboboxOption[] = [
+    { id: 'thing1', value: 'Abc Thing 1' },
+    { id: 'thing1a', value: 'Bde Thing 2' },
+    { id: 'thing1b', value: 'Bef Thing 3' },
+    { id: 'thing4', value: 'Efg Thing 4' },
+    { id: 'athing1', value: 'Abc Thing 1' },
+    { id: 'athing1a', value: 'Bde Thing 2' },
+    { id: 'athing1b', value: 'Bef Thing 3' },
+    { id: 'athing4', value: 'Efg Thing 4' },
+];
+
+const optionsB: ComboboxOption[] = [
+    { id: 'o1', value: 'Aaaaaaaaaaaaa' },
+    { id: 'o2', value: 'Abaaaaaaaaaaa' },
+    { id: 'o3', value: 'Abcaaaaaaaaaa' },
+    { id: 'o4', value: 'Abcdaaaaaaaaa' },
+    { id: 'o5', value: 'Abcdeaaaaaaaa' },
+    { id: 'o6', value: 'Abcdefaaaaaaa' },
+    { id: 'o7', value: 'Abcdefgaaaaaa' },
+    { id: 'o8', value: 'Abcdefghaaaaa' },
+    { id: 'o9', value: 'Abcdefghiaaaa' },
+    { id: 'o10', value: 'Abcdefghijaaa' },
+    { id: 'o11', value: 'Abcdefghijkaa' },
+    { id: 'o12', value: 'Abcdefghijkla' },
+    { id: 'o13', value: 'Abcdefghijklm' },
+];
```

Comment:
Note to selves, we should probably come up with a closer to real life set of demo data...